internal/protocoltest/awsrestjson/api_op_EmptyInputAndEmptyOutput_test.go

```diff
@@ -120,7 +120,7 @@ func TestClient_EmptyInputAndEmptyOutput_awsRestjson1Deserialize(t *testing.T) {
 		"RestJsonEmptyInputAndEmptyOutput": {
 			StatusCode:    200,
 			BodyMediaType: "application/json",
-			Body:          []byte(``),
+			Body:          []byte(`{}`),
 			ExpectResult:  &EmptyInputAndEmptyOutputOutput{},
```

Comment:
I think syncing with the latest version of `smithy` will update the generation of this test to be the empty body.